README.md

```diff
@@ -2,25 +2,34 @@
 
 # Neurobagel API
     
-[![Tests](https://img.shields.io/github/actions/workflow/status/neurobagel/api/test.yaml?color=BDB76B&label=test&style=flat-square)](https://github.com/neurobagel/api/actions/workflows/test.yaml)
-[![Coverage](https://img.shields.io/coverallsCoverage/github/neurobagel/api?style=flat-square&color=8FBC8F)](https://coveralls.io/github/neurobagel/api)
-[![Python](https://img.shields.io/badge/python-3.10-4682B4?style=flat-square)](https://www.python.org/)
-[![License](https://img.shields.io/github/license/neurobagel/api?color=CD5C5C&style=flat-square)](LICENSE)
+[![Tests](https://img.shields.io/github/actions/workflow/status/neurobagel/api/test.yaml?color=BDB76B&label=test&style=flat)](https://github.com/neurobagel/api/actions/workflows/test.yaml)
+[![codecov](https://codecov.io/gh/neurobagel/api/graph/badge.svg?token=ZEOGQFFZMJ)](https://codecov.io/gh/neurobagel/api)
+[![Python](https://img.shields.io/badge/python-3.10-4682B4?style=flat)](https://www.python.org/)
+[![License](https://img.shields.io/github/license/neurobagel/api?color=CD5C5C&style=flat)](LICENSE)
 
 </div>
 
 The Neurobagel API is a REST API, developed in [Python](https://www.python.org/) using [FastAPI](https://fastapi.tiangolo.com/) and [Pydantic](https://docs.pydantic.dev/).
 
 Please refer to our [**official documentation**](https://neurobagel.org/api/) for more information on how to use the API.
 
-- [Quickstart](#quickstart)
-- [Local installation](#local-installation)
-    - [Environment variables](#set-the-environment-variables)
-    - [Docker (recommended)](#docker)
+- [Neurobagel API](#neurobagel-api)
```

Comment:
🍒  How about removing this top level shortcut? I don't believe it adds much (if the user clicks it, they don't go anywhere), and it means that everything else in the table of contents has to be indented.

Reply:
Oh yeah, I'm not even sure where that's from!